### DISCUSSION TITLE
Enable / Disable Breadcrumbs

### DISCUSSION THREAD

**Discussion by newtec-foggia** [laravel/nova-issues#4138] [Ideas & Feature Requests] [answered]
can be a useful part of the UI for deeply nested admin panels

**Comment by ianrobertsFF** [comment]
I've just built my own using Cards, but its a pain as you have to maintain your own fork (or use nova-assets-changer) and modify the Update and Create views in order to support cards.

Then it makes sense to extend the Cards class and add the hideOn, onlyOn, etc that fields uses, then of course you have to modify the Index and Detail views to support that. 

Hoping one day we get this Cards on all resource views baked in, but once I'm finished with polishing I'm happy to share what I have, it will take continual maintenance to ensure compatibility with Nova though.

**Reply by ianrobertsFF** [reply]
Scratch that I've worked out how to prepend to Novas pages on the fly, should allow me to create this without needing any overrides.

**Comment by ianrobertsFF** [comment]
Breadcrumbs Package is incoming, but I've packaged up the ability to have cards on all views first, so if you'd rather roll your own breadcrumbs, you can use these to do it:

https://novapackages.com/packages/formfeed-uk/nova-resource-cards

**Comment by piljac1** [comment]
People have been asking for breadcrumbs a lot since 2018. Unfortunate that 4 years later, it is still in demand and nothing but 3rd party packages have been done to resolve a simple UX problem.

**Comment by ianrobertsFF** [comment]
Sorry this took so long guys, have been busy with other stuff and hadn't had the chance to generalise it for release.

https://novapackages.com/packages/formfeed-uk/nova-breadcrumbs

**Comment by crynobone** [comment, marked as answer]
Implemented in 4.19.0

You can enable it via the following:

```php
Nova::withBreadcrumbs();
```
